libs/logging-bundle/tests/datadogStubs.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DDTrace;
+
+// phpcs:disable
+if (function_exists('DDTrace\\current_context')) {
+    return;
+}
+// phpcs:enable
+
+function current_context(): array
+{
+    return [
```

Comment:
To nevypadá úplně jako stub

Reply:
polyfill? nevim.. proste je to neco, co lokalne v testech fejkuje tu DD funkci